src/main/java/task.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+
+public class task {
+    private int mark;
+    private String line;
+    ArrayList<String> namelist=new ArrayList<>();
+    ArrayList<Character> donelist=new ArrayList<>();
+    public task(){
+        this(0,"null");
+    }
+    public task(int mark, String line){
+        this.mark=mark;
+        this.line=line;
+    }
+    public void getList(){
+      System.out.println("Here are the tasks in your list:");
```

Comment:
Layout is recommended to be 4 spaces. So your System.out.println("...") should be 4 spaces away from the public void getList(){}